### DISCUSSION TITLE
New Form of Exploitation - Forced Criminality

### DISCUSSION THREAD

**Discussion by cmwyndham** [UNMigration/HTCDS#27] [Ideas]
As many of you will know, an emerging issue, particularly in the Southeast Asian region is people being trafficked into criminal owned compounds and forced to conduct romance and other online financial scams.  

Blue Dragon has been involved with rescuing and supporting some victims, and we are currently unable to identify them effectively in our case management database as the HTCDS doesn't include this form of exploitation. 

UNODC have recently developed indicators for the identification and understanding of this form of exploitation - see attached. 

I think that the HTCDS needs to include this form of exploitation in the standards. Not sure of the process to add something, but I would appreciate consideration of this - it will be particularly necessary for organisations working in SEA, but also important more broadly so that we can shed light on this particular issue, and understand the size of the problem, and demographics of the victims. 
[UNODC Key Indicators of TIP for Forced Criminality FINAL.pdf](https://github.com/UNMigration/HTCDS/files/12143746/UNODC.Key.Indicators.of.TIP.for.Forced.Criminality.FINAL.pdf)

**Comment by philhb** [comment]
Thanks so much for your feedback/suggestion. I've just created an issue record on this, which is the normal route to evaluate proposed updates.
